### DISCUSSION TITLE
T-Beam not decoding DFM-17

### DISCUSSION THREAD

**Discussion by n4fv** [dl9rdz/rdz_ttgo_sonde#376] [TTGO firmware]
I just got a T-Beam V 1.1 with IL9225 display going and it will not decode DFM-17 sondes.  I have tried a couple versions of the firmware and it still won't decode.  I can see the sonde signal on the spectrum display.   I can offset the RX and it shows + or - depending on which side of center frequency I offset. A couple of TTGO pax 32 boards flashed with the same computer/cable have no issue decoding DFM-17's.   Is there a setting in the config for the T-Beam other than the display that needs be reset?

**Comment by hr-ru** [comment]
There should be not difference with regard of the decoding process on T-Beam vs. other TTGO boards.
- There are some boards sold as "T-Beam V1.1" which are actually V1.2 boards with different power management and LoRa chip. These boards would not work at all (the power management chip is not supported by my software [I would have a patch for that], and the more recent LoRa chip is not supported as well). I rule out this problem in your case, as it seems that the software is running fine, even displaying the spectrum.
- There is one issue that happens on T-Beam boards with ILI9225 display that you do not have on the boards with integrated LCD display: Interference caused by the display bus. If you have an antenna far away, separated from your T-Beam by a cable, this should be a non-issue. If you are using the small antenna directly connected to the T-Beam board, this may be an issue. How severe it is depends on the cabling. Shorter cables, or shielded cables (warp them with aluminium foil) may help reducing this interference. (This problem should affect all sondes, but for example RS41 have some gaps in transmissions and the display update is mainly done in that gap, so you may this effect more severly on DFM sondes than on RS41).  But still strong enough signals should just decode fine, the problem is "just" a reduced sensitivity, as the interference raises the noise leve a bit.
- You might slightly mitigate the interference by slightly reducing the SPI frequency (the default 40 MHz has a harmonic on 400 MHz), 38 MHz or 39 MHz might be better.
No other ideas on what might cause the problem...

**Reply by n4fv** [reply]
I tried resetting the SPI to 38 and 39 mhz.  Still no change.  I think something may be wrong with the board because I tried a retrieved DFM-17 with the antenna removed and terminated into a 50 ohm resistor about 20 feet away.  It still doesn't decode it.  I think I will try changing the display and see if that is the issue. Thanks for answering.

**Comment by DadoRdz** [comment]
Unfortunately, the recently acquired module is actually Lilygo T-Beam V1.2 with PMU chip AXP2101 (2023 version), instead of AXP192 which was on T-Beam V1.1. As far as I understand, the current RDZ firmware does not support this module, only the old one. Is there a plan to create and publish a new firmware that will be compatible with this module? Is the LORA receiver on this module usable at all for the RDZ receiver? Should I wait for the release of the new Firmware or should I rather return this module and try to get the old one?

**Reply by dl9rdz** [reply]
There is some support in the recent devel version, see the discussion here: https://github.com/dl9rdz/rdz_ttgo_sonde/pull/379

Everything should be working with that version, except for the battery info display. There is a pull request for that (see https://github.com/dl9rdz/rdz_ttgo_sonde/pull/381) that I still have to merge (but if you prefer you could also compile that version yourself).

The LoRa chip in the T-Beam V1.2 is the "old" one, so that should be fine.

**Comment by DadoRdz** [comment]
Thank you very much for very prompt and useful answer. I will try the dev
version. Battery status indication is not particular important for me.

Best regards,

Dado
